src/eval-frame/components/panes/onboarding-content.jsx

```diff
@@ -92,21 +92,21 @@ export default class OnboardingContent extends React.Component {
           <Element key="examples">
             <ElementTitle>Start with a Template</ElementTitle>
             <ElementBody>
-              <ElementBlockLink href="https://extremely-alpha.iodide.io/tryit">
+              <ElementBlockLink href="https://iodide.io/tryit">
```

Comment:
We should just not include the URLs here at all -- we can just use a relative path to the resource. e.g. `<ElementBlockLink href="/tryit">` should work just fine.

Reply:
ah thank you! yes, much better idea, i'll make those changes

Reply:
hmmm, this actually may not be the best way to do this in the short term: it will break the demo links on eg the internal mozilla instance. i think we should keep the links to `iodide.io` for now, and we can revisit this when we eventually get around to a better system for doing featured notebooks.